incubator/perfsonar-testpoint-demo/images/run-my-tests.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+run_test(){
+echo "======= Running '$1' =======" | tee -a /var/log/demo/demo.out /proc/1/fd/1
+$1 | tee -a  /var/log/demo/demo.out /proc/1/fd/1
+echo -e "\n" | tee -a /var/log/demo/demo.out /proc/1/fd/1
+}
+
+run_all_tests_to() {
+
+#run_test "pscheduler troubleshoot"
+run_test "pscheduler task throughput -t 30 --dest $1"
+run_test "pscheduler task latency --packet-count 6000 --packet-interval .01 --dest $1"
+run_test "pscheduler task --tool tracepath trace --dest $1"
+
+}
+
+#Waiting for a limited time for services to start up and be ready
+sleep 20
+i="0"
+pscheduler troubleshoot
+while [ $? -ne 0 ] && [ $i -le 4 ]
+do
+echo "Local perfSONAR services are not ready yet...waiting for 15 secs before checking again...." | tee -a /var/log/demo/demo.out /proc/1/fd/1
```

Comment:
```suggestion
echo "Local perfSONAR services are not ready yet...waiting for 15 secs before checking again...."
```
Could simplify here by only sending output to `stdout`

Reply:
I think this is dependent on the first suggested change. With supervisord running as a non daemon process, sending to stdout does not capture the needed output.

Reply:
same as above